task.go

```diff
@@ -200,7 +200,7 @@ func (e *Executor) runCommand(ctx context.Context, t *taskfile.Task, call taskfi
 
 	switch {
 	case cmd.Task != "":
-		return e.RunTask(ctx, taskfile.Call{Task: cmd.Task, Vars: cmd.Vars})
+		return e.RunTask(ctx, taskfile.Call{Task: cmd.Task, Vars: cmd.Vars, IgnoreError: cmd.IgnoreError || call.IgnoreError})
```

Comment:
Do we really want to delegate this options to tasks being called, too? I think I'd expect `ignore_error: true` to be valid to the *current* task only

Reply:
Related to @smyrman  answer

Reply:
```go
case cmd.Task != "":
	var err error
	if err := e.RunTask(ctx, taskfile.Call{Task: cmd.Task, Vars: cmd.Vars}); !cmd.IgnoreError {
		return err
	}
	return nil